src/content/blog/whats-new-february-2024.mdx

```diff
@@ -0,0 +1,208 @@
+---
+title: "What's new in Astro - February 2024"
+description: "We started of 2024 strong"
+publishDate: "February 29, 2024"
+authors:
+  - elian
+coverImage: "/src/content/blog/_images/whats-new-shared.webp"
+socialImage: "/src/content/blog/_images/whats-new-february-2024/og.webp"
+lang: "en"
+---
+import ShowcaseGrid from './_whats-new-components/ShowcaseGrid.astro';
+import ThemeGrid from './_whats-new-components/ThemeGrid.astro';
+
+little sentence here @todo:
+
+## Updates from Mission Control
+
+- We have a [new channel on Discord](/chat) `#events`, where you can share events happening, share when you're live, and more.
+
+## Releases
+
+We're getting things ready for the launch of [Astro Studio](https://studio.astro.build).
+
+- [Astro 4.3](https://astro.build/blog/astro-430/)
+- [Astro 4.4](https://astro.build/blog/astro-440/) 
+- [Starlight 0.18](https://github.com/withastro/starlight/releases/tag/%40astrojs%2Fstarlight%400.18.0) 
+- [Starlight 0.19](https://github.com/withastro/starlight/releases/tag/%40astrojs%2Fstarlight%400.19.0)
+- [Starlight 0.20](https://github.com/withastro/starlight/releases/tag/%40astrojs%2Fstarlight%400.20.0)
+
+## Docs
+
+This month Docs added two new guide pages to docs on adding [authentication](https://docs.astro.build/en/guides/authentication/) and [e-commerce](https://docs.astro.build/en/guides/ecommerce/) options to your Astro project! These are two popular features that people ask about, and we wanted to provide some basic guidance to get you started.
+
+Our weekly Discord call “Talking and Doc’ing” is a chance for you to watch Team Docs work in public, jump in a shared Codespace or Gitpod Workspace, and contribute to the docs live, together. This month’s topics included:
+
+- Part two of an epic PR to make docs URL paths consistent with the locations in the sidebar settings that included:
+	- 📅 2 weeks.
+	- 🕓 4 hours of live streaming.
+	- 🧑‍🤝‍🧑 10 people in the same shared codebase at once
+	- 🗃 678 files changed.
+	- ✅ ALL CHECKS PASSED.
+
+- figuring out what a Starlight `<Steps />` component might look like. Just an ordered list? We think not!
+
+- exploring building in Astro's native i18n routing features to the default Starlight template.
```

Comment:
@ElianCodes I will need you to figure out why my bullet points are weird!  :joy: